docs/docs/language_concepts/02_control_flow.md

```diff
@@ -19,7 +19,7 @@ for i in 0..10 {
 };
 ```
 
-The index for loops is of type `u64`.
+The index for loops is of type `field`.
```

Comment:
We changed this to be u64 -- do you see something different when you run Noir programs?

Reply:
@kevaundray  oh i see, `8` is of type `field`, this code works like this : 
`if (i+j < 8 as u64) {...}`

my bad!

Reply:
It seems that `8` is automatically casted to a `u64` only if we explicitly declare `i` and `j` as `u64` in the comparison.

Reply:
can you open an issue for this?

Reply:
Sure. I'll do it tomorrow in the morning as soon as i am available. 
Should i close this PR as it is finally irrelevant ?